config.m4

```diff
@@ -41,6 +41,8 @@ if test "$PHP_PDO_SNOWFLAKE" != "no"; then
   LDFLAGS="$LDFLAGS $SNOWFLAKE_CLIENT_DIR/deps-build/linux/curl/lib/libcurl.a"
   LDFLAGS="$LDFLAGS -Wl,--no-whole-archive"
 
+  CFLAGS="-std=c99 $CFLAGS"
```

Comment:
Is C99 required to get the PDO Driver to build? Otherwise I think this should be C89 or C90.

Reply:
Yes, variable parameter macro requires C99. Actually mysql did. Will remove this if PDO won't be built with other platform.

Reply:
Ah, interesting